docs/Security-Compliance/IR/Sys-Admin-Acct-Mgmt.md

```diff
@@ -0,0 +1,149 @@
+# System Admin Account Restoration Strategy
+
+## Scenario: Sys Admin Account(s) Compromised
+
+**TDP has 2 sys admins, and one has turned evil :smiling_imp: and decided to compromise the other's account such that they can no longer access TDP DAC. Below is a screenshot of the compromised account:**
+
+_Sys admin account (i.e. `alexandra.pennington@acf.hhs.gov`)  has been modified with a different username/email address, first name, and last name._
+![](https://i.imgur.com/416KxSy.png)
```

Comment:
You can insert this inline via github so we never lose an outside link to another service.